src/Microsoft.AspNetCore.Razor.Tools/GenerateCommand.cs

```diff
@@ -189,7 +189,7 @@ protected override bool ValidateArguments()
                 }
             }
 
-            return success ? 0 : -1;
+            return success ? 0 : 1;
```

Comment:
Can we make this a more unique code? We're using this to specifically signal a certain kind of output to DotnetToolTask and 1 is meant to be general purpose error. I'd say, pick 65 (from https://stackoverflow.com/a/1535733)